tests/unit/Stackdriver/SpanConverterTest.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * Copyright 2018 OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Tests\Unit\Trace\Exporter\Stackdriver;
+
+use OpenCensus\Trace\Exporter\Stackdriver\SpanConverter;
+use OpenCensus\Trace\Annotation as OCAnnotation;
+use OpenCensus\Trace\Link as OCLink;
+use OpenCensus\Trace\MessageEvent as OCMessageEvent;
+use OpenCensus\Trace\Span as OCSpan;
+use OpenCensus\Trace\Status as OCStatus;
+use Prophecy\Argument;
+use Google\Cloud\Trace\Link;
+use Google\Cloud\Trace\Span;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group trace
+ */
+class SpanConverterTest extends TestCase
+{
+    /**
+     * @var SpanData[]
+     */
+    private $spans;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->spans = array_map(function ($span) {
+            return $span->spanData();
+        }, [
+            new OCSpan([
+                'name' => 'span',
+                'startTime' => microtime(true),
+                'endTime' => microtime(true) + 10
+            ])
+        ]);
+    }
+
+    public function testFormatsTrace()
+    {
+        $span = new OCSpan([
+            'name' => 'span',
+            'startTime' => microtime(true),
+            'endTime' => microtime(true) + 10
+        ]);
+        $span = SpanConverter::convertSpan($span->spanData());
+
+        $this->assertInstanceOf(Span::class, $span);
+        $this->assertInternalType('string', $span->name());
+        $this->assertInternalType('string', $span->spanId());
+        $this->assertNull($span->parentSpanId());
+        $this->assertRegExp('/\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{9}Z/', $span->jsonSerialize()['startTime']);
```

Comment:
better to have a constant for the regex?

Reply:
Makes sense - done.